distributions/otelcol-contrib/otelcol-contrib.service

```diff
@@ -5,6 +5,7 @@ After=network.target
 [Service]
 EnvironmentFile=/etc/otelcol-contrib/otelcol-contrib.conf
 ExecStart=/usr/bin/otelcol-contrib $OTELCOL_OPTIONS
+ExecReload=/bin/kill -HUP $MAINPID
```

Comment:
please also add this to otelcol and otelcol-otlp

Reply:
@mowies Done :-)